release/src/main/groovy/run_release_candidate_python_quickstart.sh

```diff
@@ -179,12 +111,13 @@ gpg --verify $ASC_FILE_NAME $BEAM_PYTHON_SDK
 
 print_separator "Creating new virtualenv and installing the SDK"
 virtualenv temp_virtualenv
-. temp_virtualenv/bin/activate && pip install $BEAM_PYTHON_SDK[gcp]
+. temp_virtualenv/bin/activate
```

Comment:
You do not need to create a virtualenv before gcloud update. virtualenv is only meaningful for the pip install that is happening a few lines below. (Maybe gcloud can move before the installing the SDK section.)

Reply:
Actually, we do need to setup virtualenv first. Since we are not permitted to modify $PATH on Jenkins hosts, we are not able to install new gcloud sdk. And virtualenv creates an isolated python environment that allows us to update gcloud there. 
In this quarter, we have a task to update gcloud on Jenkins nodes so that we don't need to do that in every validation run.

Reply:
Thank you for the explanation.